languages/Language.php

```diff
@@ -687,6 +687,32 @@ public static function getLanguageNames( $customisedOnly = false ) {
 	}
 
 	/**
+	 * Return languages supported by fandom
+	 *
+	 * @return array
+	 */
+	public static function getSupportedLanguages() {
+		return [ 'de', 'en', 'es', 'fr', 'it', 'ja', 'pl', 'pt-br', 'ru', 'zh', 'zh-tw' ];
```

Comment:
Let's put each item in a separate line. It will be easier to track changes (i.e. run `git blame`) as we add more languages to the list.